### DISCUSSION TITLE
i can't download it

### DISCUSSION THREAD

**Discussion by GergesBernaba1** [community/community#17392] [Other Feature Feedback, Questions, & Ideas]
when i try to download and install it on visual studio latest version 

this error occurs 

------------------------------------------------------

27/05/2022 05:34:33  - Microsoft VSIX Installer
27/05/2022 05:34:33  - -------------------------------------------
27/05/2022 05:34:33 - vsixinstaller.exe version:
27/05/2022 05:34:33  - 17.2.2174
27/05/2022 05:34:33- -------------------------------------------
27/05/202- Command line parameters:
27/05/2022 05:34:33  - C:\Program Files (x86)\Microsoft Visual Studio\Installer\resources\app\ServiceHub\Services\Microsoft.VisualStudio.Setup.Service\VSIXInstaller.exe,D:\Downloads File\GitHub.Copilot.Vsix.1.14.0.1.vsix
 - Microsoft VSIX Installer
27/05/2022 05:34:33 - -------------------------------------------
27/05/2022 05:34:34  - System.NullReferenceException: Object reference not set to an instance of an object.
   at VSIXInstaller.SetupExtensions.GetLaunchableInstances(IQuery query)
   at VSIXInstaller.SupportedSKUs.EnumerateIsolatedInstalls(Action`1 callback)
   at VSIXInstaller.SupportedSKUs.AddInstalledLocationBasedSKUs(IntPtr userToken)
   at VSIXInstaller.SupportedSKUs.InitializeSupportedSKUs(IntPtr userToken)
   at VSIXInstaller.ExtensionService.InitializeSupportedSKUs(ICommandLineData cmdLineData, IntPtr duplicatedUserToken)
   at VSIXInstaller.App.Initialize(Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize()
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)

**Comment by brunoniconeves** [comment]
I've the same issue here

06/12/2022 07:57:26 - Microsoft VSIX Installer
06/12/2022 07:57:26 - -------------------------------------------
06/12/2022 07:57:26 - vsixinstaller.exe version:
06/12/2022 07:57:26 - 17.4.2119
06/12/2022 07:57:26 - -------------------------------------------
06/12/2022 07:57:26 - Command line parameters:
06/12/2022 07:57:26 - C:\Program Files (x86)\Microsoft Visual Studio\Installer\resources\app\ServiceHub\Services\Microsoft.VisualStudio.Setup.Service\vsixinstaller.exe,/appidinstallpath:C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\devenv.exe,/skuName:Community,/skuVersion:17.4.33122.133,/appidname:Microsoft Visual Studio Community 2022,/culture:en-US,/noep,C:\Users\Bruno\AppData\Local\Temp\VSIXa5snvkx0.vsix
06/12/2022 07:57:26 - -------------------------------------------
06/12/2022 07:57:26 - Microsoft VSIX Installer
06/12/2022 07:57:26 - -------------------------------------------
06/12/2022 07:57:29 - System.NullReferenceException: Object reference not set to an instance of an object.
   em VSIXInstaller.SetupExtensions.GetLaunchableInstances(IQuery query)
   em VSIXInstaller.SupportedSKUs.EnumerateIsolatedInstalls(Action`1 callback)
   em VSIXInstaller.SupportedSKUs.AddInstalledLocationBasedSKUs(IntPtr userToken)
   em VSIXInstaller.SupportedSKUs.InitializeSupportedSKUs(IntPtr userToken)
   em VSIXInstaller.ExtensionService.InitializeSupportedSKUs(ICommandLineData cmdLineData, IntPtr duplicatedUserToken)
   em VSIXInstaller.App.Initialize(Boolean isRepairSupported)
   em VSIXInstaller.App.Initialize()
   em System.Threading.Tasks.Task`1.InnerInvoke()
   em System.Threading.Tasks.Task.Execute()

**Comment by brunoniconeves** [comment]
I found the solution for this issue:

Since VSIXInstaller will query all instance before it install an extension, if one instance throws it will directly fail.

To unblock this case, you need delete folder: “%ProgramData%\Microsoft\VisualStudio\Packages\_Instances\XXXXXXX” and then try to install extensions again.

I deleted only the oldest ones.
